packages/core/src/http-client/csrf-token-header.ts

```diff
@@ -58,7 +62,10 @@ function makeCsrfRequest<T extends HttpRequestConfig>(
   return executeHttpRequest(destination, appendSlash(axiosConfig))
     .then(response => response.headers)
     .catch(e1 => {
-      logger.error(
+      if (hasCsrfToken(e1)) {
+        return e1.response.headers;
+      }
+      logger.warn(
         new ErrorWithCause(
           'Initial try to fetch CSRF token failed - retry without slash at ',
```

Comment:
I know this was here before, but it seems that there is a part of the sentence missing?
```suggestion
          'First attempt to fetch CSRF token failed. Retrying without trailing slash.',
```